docs/Quickstart.md

```diff
@@ -17,71 +19,85 @@ $ git commit -m 'new rails app'
 
-cli:
+CLI:
 ```sh
 $ heroku create name
 $ heroku git:remote -a name
 ```
 
-Now we need to let git know where the remote server is so we'll be able to deploy later
+Once you have created an app on Heroku, we need to let Git know where the Heroku server is so we can deploy to it later. Copy the app's name from your Heroku dashboard and substitute `appname.git` with the name you chose earlier:
 
 web:
 ```sh
 # https://dashboard.heroku.com/new
-$ git remote add heroku git@heroku.com:appinfive.git
+$ git remote add heroku git@heroku.com:appname.git
 ```
 
 3. Create a new App in the Shopify Partner dashboard
 -----------------------------------------
 * Create a Shopify app in the [Partners dashboard](https://partner.shopify.com). For this tutorial, you can choose either a public or custom app, but you can [learn about App Types here.](https://help.shopify.com/en/manual/apps/app-types)
 [https://app.shopify.com/services/partners/api_clients](https://app.shopify.com/services/partners/api_clients)
-* Set the callback url to `https://<name>.herokuapp.com/`
+* Set the callback url to `https://<appname>.herokuapp.com/`
 * Choose an embedded app
-* Set the app's `redirect_uri` to `https://<name>.herokuapp.com/auth/shopify/callback`
-
+* Set the app's `redirect_uri` to `https://<appname>.herokuapp.com/auth/shopify/callback`
 
-4. Add ShopifyApp to gemfile
+4. Add ShopifyApp to Gemfile
 ----------------------------
+
+Run these commands to add the `shopify_app` Gem to your app:
+
 ```sh
 $ echo "gem 'shopify_app'" >> Gemfile
 $ bundle install
 ```
 
-Note - it's recommended to use the latest version of the Shopify Gem. Check the Git tags to see the latest release and then add it to your Gemfile e.g `gem 'shopify_app', '~> 7.0.0'`
+**Note:** we recommend using the latest version of Shopify Gem. Check the [Git tags](https://github.com/Shopify/shopify_app/tags) to see the latest release version and then add it to your Gemfile e.g `gem 'shopify_app', '~> 7.0.0'`
 
 5. Run the ShopifyApp generator
 -------------------------------
+
+Generate the code for your app by running these commands: 
+
 ```sh
-# use the keys from your app in the partners area
+# Use the keys from your app you created in the partners area
 $ rails generate shopify_app --api_key <shopify_api_key> --secret <shopify_api_secret>
 $ git add .
 $ git commit -m 'generated shopify app'
 ```
 
 If you forget to set your keys or redirect uri above, you will find them in the shopify_app initializer at: `/config/initializers/shopify_app.rb`.
 
-We recommend adding a gem or utilizing ENV variables to handle your keys before releasing your app.
+We recommend adding a gem or utilizing environment variables (ENV) to handle your keys before releasing your app. [Learn more about using environment variables.](https://www.honeybadger.io/blog/ruby-guide-environment-variables/)
```

Comment:
"(ENV)" is a little ambiguous now, and we referenced the lowercase name of the file above. Maybe instead:

> ...utilizing an environment variables file (`.env`) to handle your keys...